conductor/lib/common.py

```diff
@@ -671,3 +672,23 @@ class TmpLoader(loader):
 
     with open(filepath) as f:
         return yaml.load(f, loader)  # nosec  (ignore bandit static analysis warning for not using safe_load [B506:yaml_load] )
+
+
+def sstr(object_, char_count=1000, pretty=True):
+    '''
+    Return a string representation of the given object, shortened to the given
+    char_count. This can be useful when printing/logging out data for debugging
+    purposes, but don't want an overwhelming wall of text to troll through.
+
+    pretty: bool. If true, will pretty print the object
+    '''
+    assert type(char_count) == int, "char_count must be int. Got: %r (%s)" % (char_count, type(char_count))
+    try:
+        s_str = pformat(object_) if pretty else str(object_)
+    except:
```

Comment:
Seems like we could make this: catch explicit:

`except (ValueError, TypeError):`